### PR TITLE
Add cache build flag INSTALL_KUBERNETES

### PR DESCRIPTION
**Pull request by xumia**
**- Why I did it**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: Makefile.cache [hunk 1]
```diff
@@ -69,7 +69,7 @@
 SONIC_COMMON_FILES_LIST :=  $(if $(wildcard cache.skip.common),, .platform slave.mk rules/functions Makefile.cache)
 SONIC_COMMON_FLAGS_LIST :=  $(CONFIGURED_PLATFORM) \
                             $(SONIC_DPKG_CACHE_SOURCE) $(SONIC_DEBUGGING_ON) \
-                            $(SONIC_PROFILING_ON) $(SONIC_ENABLE_SYNCD_RPC)
+                            $(SONIC_PROFILING_ON) $(SONIC_ENABLE_SYNCD_RPC) $(INSTALL_KUBERNETES)
```
Comment (lguohan) [inline Makefile.cache:72]: why this common flag? it does not affect most build packages.

Comment (lguohan) [reply Makefile.cache:72]: it is unclear why you did this, can you explain more on the motivation?

Comment (xumia) [reply Makefile.cache:72]: It is to generate a new hash for the packages with INSTALL_KUBERNETES enabled. The option only impacts the installer images, right? And there are no cache for the images which are built every time.

Comment (lguohan) [reply Makefile.cache:72]: the common flags list impact every build artifacts.

Comment (xumia) [reply Makefile.cache:72]: I think the PR is not necessary, the INSTALL_KUBERNETES is only applied to the installer images (SONIC_INSTALLERS), and we never use cache for the images.

**Pull request closed without merging**